ui/about.lua

```diff
@@ -3,48 +3,90 @@ local S = minetest.get_translator("mail")
 
 local FORMNAME = "mail:about"
 
+local groups = {
+	{ "o", S("Original author")},
+	{ "c", S("Code")},
+	{ "i", S("Internationalization")},
+	{ "t", S("Textures")},
+	{ "a", S("Audio")},
+}
+
+local contributors = {
+	{ name = "Cheapie", groups = {"o", "c"} },
+	{ name = "aBlueShadow", groups = {"c"} },
+	{ name = "APercy", groups = {"i"} },
+	{ name = "Athozus", groups = {"c", "i"} },
+	{ name = "BuckarooBanzay", groups = {"c"} },
+	{ name = "Chache", groups = {"i"} },
+	{ name = "Dennis Jenkins", groups = {"c"} },
+	{ name = "Emojigit", groups = {"i"} },
+	{ name = "Eredin", groups = {"i"} },
+	{ name = "fluxionary", groups = {"c"} },
+	{ name = "imre84", groups = {"c"} },
+	{ name = "Muhammad Rifqi Priyo Susanto", groups = {"i"} },
+	{ name = "NatureFreshMilk", groups = {"c", "t"} },
+	{ name = "Niklp", groups = {"c", "i"} },
+	{ name = "Nuno Filipe Povoa", groups = {"a"} },
+	{ name = "nyomi", groups = {"i"} },
+	{ name = "OgelGames", groups = {"c"} },
+	{ name = "Panquesito7", groups = {"c"} },
+	{ name = "Peter Nerlich", groups = {"c"} },
+	{ name = "Rubenwardy", groups = {"c"} },
+	{ name = "savilli", groups = {"c"} },
+	{ name = "Singularis", groups = {"c"} },
+	{ name = "SX", groups = {"c"} },
+	{ name = "TheTrueBeginner", groups = {"i"} },
+	{ name = "Thomas Rudin", groups = {"c"} },
+	{ name = "Toby1710", groups = {"c"} },
+	{ name = "whosit", groups = {"c"} },
+	{ name = "Wuzzy", groups = {"i"} },
+	{ name = "y5nw", groups = {"c", "i"} },
+}
+
 function mail.show_about(name)
 	local formspec = [[
 			size[10,6;]
-			tabheader[0.3,0.875;optionstab;]] .. S("Settings") .. "," .. S("About") .. [[;2;false;false]
+			tabheader[0,0;optionstab;]] .. S("Settings") .. "," .. S("About") .. [[;2;false;false]
 			button[9.35,0;0.75,0.5;back;X]
-			label[0,0.7;Mail]
-			label[0,1.1;]] .. S("Provided by mt-mods") .. [[]
-			label[0,1.5;]] .. S("Version") .. [[ : 1.4.0-dev]
-			label[0,2.0;]] .. S("Licenses") .. [[ :]
-			label[0.2,2.4;]] .. S("Expat (code), WTFPL (textures)") .. [[]
-			label[0,3.2;https://github.com/mt-mods/mail]
-			label[0,3.6;https://content.minetest.net/packages/mt-mods/mail]
-			textarea[0.5,4.8;4,5.5;;]] .. S("Note") .. [[;]] ..
+
+			box[0,0;3,0.45;]] .. mail.get_color("highlighted") .. [[]
+			label[0.2,0;Mail]
+
+			label[0.2,0.5;]] .. S("Provided by mt-mods") .. [[]
+			label[0.2,0.9;]] .. S("Version") .. [[ : 1.4.0-dev]
```

Comment:
(Nitpicking:) Not exactly relevant to this PR, but since some (e.g. east Asian) languages use the fullwidth colon `：` this would IMO make sense:

```suggestion
			label[0.2,0.9;]] .. S("Version: @1", "1.4.0-dev") .. [[
```